async-openai/src/types/mod.rs

```diff
@@ -72,3 +72,7 @@ impl From<UninitializedFieldError> for OpenAIError {
         OpenAIError::InvalidArgument(value.to_string())
     }
 }
+
+pub trait Streamable {
```

Comment:
Seems like this will become public API and not just internal to crate, in which case `Streamable` is very overloaded word and open to individual interpretation, becauses usually responses are "streamable" and not requests.

I'd suggest renaming it. Some suggestions like: `trait RequestForStream` or method to be `is_request_for_stream` or something more appropriate where its less open to interpretation.